tests/test_flow/test_persistence_random.py

```diff
@@ -371,3 +425,13 @@ def test_random_flow_varied(harness, n_bindings, n_iterations, p_persist, p_mult
         p_multi_out=p_multi_out,
     )
     RandomFlowTester(harness, config).run()
+
+
+@pytest.mark.slow
+def test_random_flow_no_tuples(harness):
```

Comment:
Since this a slow test, we probably want to remove it once we "activate" tuple descriptors. Does this sound right? If it does, we should leave a TODO to remove this test and clean up this logic.

Reply:
I agree we should rethink this once we activate tuple descriptors. I'm not certain we should totally remove it, since it's still useful to know that both `@bn.outputs` and `@bn.returns` work. But maybe we should just randomly use one or the other only when they're equivalent.